doubleml/_utils.py

```diff
@@ -150,7 +151,9 @@ def _dml_cv_predict(estimator, x, y, smpls=None,
                                      for idx, (train_index, test_index) in enumerate(smpls))
 
         preds = np.full(n_obs, np.nan)
+        targets = np.full(n_obs, np.nan)
         train_preds = list()
+        train_targets = list()
```

Comment:
As far as I see, we're not using the `train_targets` right now, right? I guess that's alright and we can leave a in-sample (fold-wise cross-validated) vs. out-of-sample (cross-fitted) evaluation for later. I'm wondering how we can implement this in a clever way... 🤔